pmoves/services/media-audio/Dockerfile

```
@@ -32,6 +32,11 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     else \
       pip install --no-cache-dir --extra-index-url https://download.pytorch.org/whl/cu124 -r requirements.txt; \
     fi
+# Diarization deps (pyannote) installed separately so they land regardless of the
+# lock-vs-requirements.txt path above — the x86_64 lock does not pin pyannote, which
+# would leave diarization broken on the GPU image (Codex #2181 P1). Full pip-compile
+# regeneration of the lock with pyannote is the ideal follow-up.
+RUN pip install --no-cache-dir pyannote.audio>=3.3.0 pydub>=0.25.1
```

Hadolint (2.14.0)

[error] Multiple redirections compete for stdout. Use cat, tee, or pass filenames instead.

(SC2261)